src/cart/crt_internal_fns.h

```diff
@@ -14,7 +14,7 @@
 /** crt_init.c */
 bool crt_initialized(void);
 
-int crt_str_to_provider(const char *provider);
+int crt_str_to_provider(char *provider);
```

Comment:
why not const ?

Reply:
I had to change it so I can update the nad_str pointer to point to the submitted provider string.